operators/benchmark-sender/run.py

```diff
@@ -0,0 +1,78 @@
+import time
+from typing import Any
+
+import numpy as np
+
+from interactem.core.logger import get_logger
+from interactem.core.models.messages import BytesMessage, MessageHeader, MessageSubject
+from interactem.operators.operator import operator
+
+logger = get_logger()
+
+# --- Global Accumulator: 256 MB ---
+ACCUMULATOR_SIZE = 268_434_432  # 256 MB in bytes
+accumulated_data = np.random.randint(0, 256, ACCUMULATOR_SIZE, dtype=np.uint8)
```

Comment:
I think we should be very clear here about the message sizes we will be sending, and allocate separate arrays for each size beforehand. Could put them into a list or dict keyed by the size, and iterate through.